sim/project/plugins.sbt

```diff
@@ -17,3 +17,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
 addSbtPlugin("org.xerial.sbt" % "sbt-pack" % "0.9.3")
 
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
+
+addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
+
+libraryDependencies += "com.github.os72" % "protoc-jar" % "3.5.1.1"
```

Comment:
what are these for?

Reply:
It's necessary because I added firrtl to the list of dependencies. If we just use the publishLocal firrtl instead, we don't need it.

Reply:
Yeah let's keep things as is.

Reply:
If there's a good reason to do this, we can move it to a separate PR.